src/App.tsx

```diff
@@ -232,6 +232,12 @@ function App(): React.ReactElement {
   }
 
   useEffect(() => {
+    const storedSessionData = sessionStorage.getItem('login')
+    if (storedSessionData !== null) {
+      console.log('Stored Value:', storedSessionData)
+    } else {
+      removeToken()
```

Comment:
@TahaKhanAbdalli - could you please rename `removeToken()` to `removeUserToken()` - since it is removing a specific token.

Reply:
Sure.